candentira/nodejs-aws-be
# Task 9: Backend For Frontend

1. Task: https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task9-bff/task.md
2. Screenshot: 
![image](https://user-images.githubusercontent.com/5624598/102022128-d0f3eb80-3d95-11eb-879f-a202f527b457.png)

3. Links:
- product-service service API endpoint URL: https://dnfpunajrc.execute-api.eu-west-1.amazonaws.com/dev/products
- example of the create product API call with all needed information: URL, payload, headers, etc.:
POST https://dnfpunajrc.execute-api.eu-west-1.amazonaws.com/dev/products
payload (raw JSON): {"price":42,"description":"test for BS","title":"ProductTestBS2","count":1}
- CART service API endpoint URL: https://i47xpdxqhk.execute-api.eu-west-1.amazonaws.com/dev/api/profile/cart
- bff-service service URL: http://candentira-bff-develop.eu-west-1.elasticbeanstalk.com
- example how to call product-service and CART services via bff-service service URL: 
http://candentira-bff-develop.eu-west-1.elasticbeanstalk.com/products
http://candentira-bff-develop.eu-west-1.elasticbeanstalk.com/cart
4. Done 13.12.2020 / deadline 13.12.2020
5. Score: 7 / 7

Evaluation criteria (each mark includes previous mark criteria)
Additional (optional) tasks
How to test:
Get products list
Create new product
Get products list - result shouldn’t have new product
Wait more than 2 minutes
Get products list - result should have new product

Please let me know if you run into any issue or would have suggestions to improve solution - I'll try to fix/improve it.
